tls/extensions/s2n_client_psk.c

```diff
@@ -26,6 +27,9 @@
 
 #define SIZE_OF_BINDER_SIZE sizeof(uint8_t)
 #define SIZE_OF_BINDER_LIST_SIZE sizeof(uint16_t)
+#define S2N_MAX_IDENTITY_SIZE 1000
+#define S2N_MAX_SECRET_SIZE 1000
+#define S2N_MAX_WIRE_INDEX 65535
```

Comment:
Looking for inputs for these values. I could probably use 65535 which is the max value for uint16_t type.

Reply:
UINT16_MAX is defined if you want to use it for 65535

Reply:
The wire index will be enforced automatically if you make it a uin16_t, identities are also only restricted to a uint16_t, and secrets are restricted by the hmac algorithm